Dokumentationen/project_management/iteration_plan_02.adoc

```diff
@@ -39,7 +39,7 @@ include::../_includes/default-attributes.inc.adoc[]
 
 Aufgaben werden in der Work Item List (https://github.com/Schmiddl99/experimenteverwaltung-i2/projects/3#column-11862065) geordnet und priorisiert.
 
-Die in dieser Iteration geplanten Aufgaben wurden in der To-Do-List dargestellt: https://github.com/Schmiddl99/experimenteverwaltung-i2/projects/3#column-11861913. Die To-Do-List verhält sich dynamisch. Deswegen sind nachfolgend die Work Items der Iteration I aufgeführt.
+Die in dieser Iteration geplanten Aufgaben wurden in der To-Do-List dargestellt: https://github.com/Schmiddl99/experimenteverwaltung-i2/projects/3#column-11861913. Die To-Do-List verhält sich dynamisch. Deswegen sind nachfolgend die Work Items der Iteration **I oder II?** aufgeführt.
 
```

Comment:
wirklich "I oder II ? " ? ich mein das klingt ein wenig komisch in einer Dokumentation :)

Reply:
II logischerweise. Muss das noch anpassen. Danke fürs hervorherben, Tim! ;)